packages/themes/README.md

```diff
@@ -58,14 +58,16 @@ Inline theming can be done by using the mixin. For example:
 // Use the default white theme here
 
 .my-dark-theme {
-  @include theme.theme(themes.$g90) {
-    // Use the dark theme here
+  @include theme.theme(themes.$g90)
+  &__something {
+    //any theme references in here, or in any nested classes/components will resolve to g90
   }
 }
```

Comment:
Can we just do 

```scss
.my-dark-theme {
  @include theme.theme(themes.$g90);
}
```

https://stackblitz.com/edit/github-e91zc8?file=src%2Findex.scss,src%2FApp.jsx,package.json